appyters/Augmented_Pathway_Enrichment_Analysis/Augmented_Pathway_Enrichment_Analysis.ipynb

```diff
@@ -576,9 +654,15 @@
     "    filtered_df = sorted_df.iloc[:display_topk]\n",
     "    if len(filtered_df) != 0:\n",
     "        display(HTML(filtered_df.to_html(index = False)))\n",
-    "        display(Markdown(f\"*Table {k}. Enrichment analysis results of {lib.replace('_', ' ')}*\"))        \n",
+    "        display(Markdown(f\"*Table 9. Enrichment analysis results of {lib.replace('_', ' ')}*\"))        \n",
```

Comment:
There is no way these figure numbers are coming out in order.. just use a global counter I guess